generator/uscore/uscore_generator.rb

```diff
@@ -231,7 +243,8 @@ def create_interaction_test(sequence, interaction)
         interaction_test = {
           tests_that: "#{sequence[:resource]} #{interaction[:code]} resource supported",
           index: sequence[:tests].length + 1,
-          link: 'https://build.fhir.org/ig/HL7/US-Core-R4/CapabilityStatement-us-core-server.html'
+          link: 'https://build.fhir.org/ig/HL7/US-Core-R4/CapabilityStatement-us-core-server.html',
+          description: "All servers #{interaction[:expectation]} make available #{interaction[:code]} interactions on #{sequence[:resource]}"
```

Comment:
How about `support` rather than `make available`? Or maybe just `...support the [RESOURCE] [INTERACTION] interaction`.

Reply:
Fixed.